contracts/starknet/L1AuthMock.cairo

```diff
@@ -4,122 +4,89 @@
 from starkware.cairo.common.cairo_builtins import HashBuiltin
 from starkware.cairo.common.math import assert_nn
 from starkware.cairo.common.hash import hash2
-from starkware.starknet.common.syscalls import (get_caller_address, get_block_number, get_block_timestamp)
+from starkware.starknet.common.syscalls import (
+    get_caller_address, get_block_number, get_block_timestamp)
 
-
-#Demo contract to test voting on snapshot X via the L1 voting contract. 
-#aside from the view functions, this contract cannot be called directly, but only via the methods in the L1 voting contract. 
-#No Double voting prevention 
+# Demo contract to test voting on snapshot X via the L1 voting contract.
+# aside from the view functions, this contract cannot be called directly, but only via the methods in the L1 voting contract.
+# No Double voting prevention
 
 const L1_VOTING_CONTRACT = (0x1234)
 
-#mapping stores 1 if the proposal has been initialized, otherwise 0
-@storage_var 
+# mapping stores 1 if the proposal has been initialized, otherwise 0
+@storage_var
 func proposal_id_store(proposal_id : felt) -> (bool : felt):
 end
 
-#double mapping that stores a counter for each vote type (1,2,3) for every proposal
+# double mapping that stores a counter for each vote type (1,2,3) for every proposal
 @storage_var
 func choices_store(proposal_id : felt, choice : felt) -> (num : felt):
-end 
-
-#event emitted after each proposal is created
-@event 
-func proposal_created_from_L1(proposal_id : felt, proposer_address : felt, block_number : felt, block_timestamp : felt):
 end
 
-#event emitted after ech vote is received 
-@event 
-func vote_received_from_L1(proposal_id : felt, voter_address : felt, choice : felt, block_number : felt, block_timestamp : felt):
-end 
+# event emitted after each proposal is created
+@event
+func proposal_created_from_L1(
+        proposal_id : felt, proposer_address : felt, block_number : felt, block_timestamp : felt):
```

Comment:
Formating is weird in that contract, is this what you get with Cairo linter? The intend doesnt make sense to me, the parentheses start and end at weird position. Maybe there is things we can define in the linter to improve that formating?

Reply:
I think this is just standard `cairo-format` run (correct me if I'm wrong @Orland0x ). I would be against defining some special linting cases specific to our repo, but maybe bring it up to the StarkWare team would be a good idea?

Reply:
yeah i didnt do anything, thats just the linter